assets/js/modules/index.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Common Module tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import path from 'path';
+
+/**
+ * Internal dependencies
+ */
+import FIXTURES from '../googlesitekit/modules/datastore/fixtures.json';
+import { convertArrayListToKeyedObjectMap } from '../util/convert-array-to-keyed-object-map';
+
+const directories = ( relativePath ) => fs.readdirSync( path.join( __dirname, relativePath ) )
+	.filter( ( name ) => fs.lstatSync( path.join( __dirname, relativePath, name ) ).isDirectory() );
+
+const getComponentNames = ( componentPath ) => fs.readdirSync( componentPath )
+	.filter( ( name ) => ! /^index|\.test\.js$/.test( name ) )
+	.map( ( name ) => name.replace( /\..*/, '' ) );
+
+describe( 'all modules', () => {
+	const fixturesKeyValue = convertArrayListToKeyedObjectMap( FIXTURES, 'slug' );
+
+	beforeAll( () => {
+		global._googlesitekitLegacyData = { modules: fixturesKeyValue };
```

Comment:
Rather than set this global here (which is a bit odd that this is needed, but I understand), let's fill out the existing global in the setup file instead since this seems to be about undefined keys in the object rather than the data itself.

https://github.com/google/site-kit-wp/blob/291eaa3c1fdaf3fad788b8e6c4077eb95b7e655e/tests/js/setup-globals.js#L27-L34

Also, this global has a bit more data in the structure than is present in the datastore fixture which represents the REST response from the `core/modules/data/list` datapoint so it might not be sufficient to simply drop-in.
![image](https://user-images.githubusercontent.com/1621608/92025262-eeadbf80-ed67-11ea-9303-d2b9b2e0ac99.png)
Things like the module's `screenID` which is present in the setup file now aren't in the response fixture.

Since there isn't a complete fixture available for this value, let's just fill in the gaps where needed for now.